astropath/bayesian.py

```diff
@@ -0,0 +1,190 @@
+"""Methods related to Bayesian association analysis"""
+
+import numpy as np
+
+import copy
+
+from astropy import units
+from astropy.coordinates import SkyCoord
+
+from IPython import embed
+
+
+def raw_prior_Oi(Pchance, Sigma_m, method, half_light=None):
+    """
+    Raw prior for a given set of Pchance values and/or n(m)
+
+    Args:
+        Pchance (float or np.ndarray):
+            Chance probability
+        Sigma_m (float or np.ndarray):
+            Number density of sources on the sky brighter than m
+        method (str):
+            inverse :: Assign inverse to Sigma_m
+            inverse1 :: Assign inverse to Sigma_m * half_light
+            inverse2 :: Assign inverse to Sigma_m * half_light**2
+            orig_inverse :: Assign inverse to P_chance
```

Comment:
I think it should be renamed to `pchance_inverse` to clearly differentiate it from the other `inverse` methods which don't use pchance

Reply:
done

Reply:
I've removed it outright